torch/csrc/autograd/variable.h

```diff
@@ -109,7 +109,12 @@ struct TORCH_API Variable : public at::Tensor {
   /// gradients. NOTE: `data` must *not* be a `Variable` already. Its dynamic
   /// type *must* be `Tensor`.
   friend Variable make_variable(
-      at::Tensor data,
+      const at::Tensor& data,
+      bool requires_grad,
+      bool allow_tensor_metadata_change);
+
+  friend Variable make_variable(
```

Comment:
We should add a comment here saying why this factory function is faster and when we can use it.